build-system/common/utils.js

```diff
@@ -145,6 +147,12 @@ function getFilesToCheck(globs, options = {}, ignoreFile = undefined) {
       log(green('INFO: ') + 'No files to check in this PR');
       return [];
     }
+    /** forbidden-terms always needs to be checked. */
```

Comment:
This statement is not exactly accurate. It only needs to be added to the list when `--local_changes` is used. Maybe delete in favor of the updated JSDoc?